pb/get_configuration.go

```diff
@@ -145,9 +146,25 @@ func (c *RemoteProvisioning) Validate() error {
 		if err := c.GetUserAgent().Validate(); err != nil {
 			return fmt.Errorf("userAgent.%w", err)
 		}
-		if c.GetCertificateAuthorities() == "" {
-			return fmt.Errorf("certificateAuthorities('%v')", c.GetCertificateAuthorities())
+		if len(c.GetCaPool()) == 0 {
+			break
 		}
+		var certificateAuthorities string
+		for i, ca := range c.GetCaPool() {
```

Comment:
SC now complains that this function is now too complex

Reply:
Fixed :)